examples/chart/teleport-cluster/templates/config.yaml

```diff
@@ -95,17 +95,41 @@ data:
     {{- toYaml .Values.labels | nindent 8 }}
   {{- end }}
     proxy_service:
+  {{- if .Values.publicAddr }}
+      public_addr: '{{ .Values.publicAddr }}'
```

Comment:
I really don't like relying on the users putting raw YAML in a YAML string, this is going to explode so hard (like YAML parsing low ports as hexa). All the other solutions seem too complex, the only viable alternative would be to use lists everywhere. :/

Reply:
Yeah, the more I think about this, the more I prefer to change all fields to lists. We don't really have to be backward compatible here as the feature was not in the chart before. It will be easier to understand and use for the users.

Reply:
I totally agree that raw YAML sucks and think using lists throughout would work better!

Reply:
Fixed